person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  private
+
+  def of_age
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] `can_use_services?` method should be made accessible to some other classes. Placing it under the `private` access modifier also makes it private. Let's improve upon this aspect by placing the `can_use_services?` above the `private` method.